code/script.js

```diff
@@ -1,53 +1,200 @@
 // DOM selectors (variables that point to selected DOM elements) goes here 👇
-const chat = document.getElementById('chat')
+const chat = document.getElementById('chat');
+const form = document.getElementById('name-form');
+const nameInput = document.getElementById('name-input');
+const inputWrapper = document.querySelector('.input-wrapper'); // Wrapper for the input field
 
-// Functions goes here 👇
+// Show album options function
+const showAlbumOptions = (name) => {
+  showMessage(`Hello ${name}, welcome to Weverse! Which BTS album would you like to order?`, 'bot');
 
-// A function that will add a chat bubble in the correct place based on who the sender is
+  // Hide the input form and show the album options
+  form.style.display = 'none'; // Hide the input field and send button
+
+  // Add album option buttons
+  chat.innerHTML += `
+    <section class="bot-msg">
+      <div class="bubble bot-bubble">
+        <button id="golden-btn">Golden</button>
+        <button id="muse-btn">Muse</button>
+        <button id="layover-btn">Layover</button>
```

Comment:
I think all these buttons that the user should click, would fit better in the inputWrapper 👀